datanode/gateway/graphql/schema.graphql

```diff
@@ -5926,10 +5985,12 @@ enum TransferDirection {
   ToOrFrom
 }
 
+union ProposalNode = Proposal | BatchProposal
+
 "Edge type containing the proposals and cursor information returned by a ProposalsConnection"
 type ProposalEdge {
   "The proposal data"
-  node: Proposal!
+  node: ProposalNode!
```

Comment:
We should do a proper deprecation here and have both `Proposal` and `ProposalNode` fields, but mark `Proposal` as deprecated.